src/rooms/dtos/requests/query-room.dto.ts

```diff
@@ -18,14 +28,28 @@ export class QueryRoomDto {
     return value;
   })
   @IsBoolean()
+  @ApiProperty({
+    description: '방 비공개 여부',
+    required: false,
+  })
   isPrivate?: boolean;
 
   @IsOptional()
   @IsEnum(RoomStatus)
+  @ApiProperty({
+    description: '방 상태',
+    enum: RoomStatus,
+    required: false,
+  })
   status?: RoomStatus;
 
   @IsOptional()
   @Type(() => Number)
   @IsInt()
+  @Min(1)
+  @ApiProperty({
+    description: '페이지네이션 커서 (마지막으로 불러온 방 ID)',
+    required: false,
+  })
```

Comment:
개인적으로 request validation을 한다면 문서에 포함하는것이 좋다고 생각하기 때문에 minLength를 추가하는건 어떻게 생각하실까요??
문서에 없는데 에러가 발생하는 경우 한번씩 문의가 있을 수 있어요. 관리하는 입장에서는 귀찮은 일이기는 하지만요

```suggestion
  @ApiProperty({
    description: '페이지네이션 커서 (마지막으로 불러온 방 ID)',
    required: false,
    minLength: 1,
  })
```